Content.Server/Audio/Jukebox/JukeboxSystem.cs

```diff
@@ -91,19 +144,19 @@ private void OnJukeboxStop(Entity<JukeboxComponent> entity, ref JukeboxStopMessa
 
     private void Stop(Entity<JukeboxComponent> entity)
     {
+        entity.Comp.TimeWhenSongEnds = null;
         Audio.SetState(entity.Comp.AudioStream, AudioState.Stopped);
+        var state = GetUiState(entity);
+        _userInterfaceSystem.SetUiState(entity.Owner, JukeboxUiKey.Key, state);
         Dirty(entity);
     }
```

Comment:
As above using a BUI state is pointless for jukebox.

Reply:
removed